src/main/resources/update/gitbucket-core_4.35.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<changeSet>
+  <addColumn tableName="WEB_HOOK">
+    <column name="HOOK_ID" type="int" nullable="false" autoIncrement="true" unique="true"/>
+  </addColumn>
+  <dropForeignKeyConstraint constraintName="IDX_WEB_HOOK_EVENT_FK0" baseTableName="WEB_HOOK_EVENT"/>
+  <dropPrimaryKey tableName="WEB_HOOK" constraintName="IDX_WEB_HOOK_PK"/>
```

Comment:
Need to drop `IDX_WEB_HOOK_PK` before adding `HOOK_ID` column?

Reply:
@takezoe Thanks for your advice! Finally test passed...
Needed unique constraint for ex. primary key (3 columns) to make foreign key for Postgres.

Reply:
Oh, I see. Thansk for dealing with it.